fractal-puppy/Engine
# Player damage improvement

This PR contains:
- New "damage" variable for each skill, this variable is multiplied for the base damage to compute the total damage.
- Stats are increased on level up. (The values can be changed from the ExperienceControllerScript).
- Machete Rain hitbox fix (now it deals damage).
- Skill Tree shows real information about skills.

**To test:**
1. Play the game.
2. Level up and se if the stats are increased.
3. Check that each skill inflicts different amounts of damage instead of 1.
4. Use MacheteRain and check that it damages enemies.
